lib/morandi/version.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module Morandi
-  VERSION = '0.12.1'.freeze
+  VERSION = '0.12.2'
```

Comment:
probably this should still be frozen

Reply:
rubocop complained about me changing this - presumably its unnecessary give the frozen_string_literal magic comment at the top of the file
<img width="761" alt="Screenshot 2020-12-21 at 13 17 45" src="https://user-images.githubusercontent.com/5340624/102781180-1ebcb500-438f-11eb-8f38-301199950595.png">